### PR TITLE
Update worker and api images

### PR DESCRIPTION
**Pull request by pbortlov**
* bump a version of OPM to 1.18.0
* install podman by dnf
* upgrade pip, grpcurl, operator-sdk
* remove builder image from Dockerfile-workers

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -96,6 +96,7 @@ services:
     environment:
       IIB_DEV: 'true'
       REQUESTS_CA_BUNDLE: /etc/pki/tls/certs/ca-bundle.crt
+      GODEBUG: 'x509ignoreCN=0'
```
Comment (midnightercz) [inline docker-compose.yml:99]: proper fix for this is to regenerate certificate with SANs instead of CN

**Review by midnightercz — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (pbortlov) [reply docker-compose.yml:99]: Yeah, I see. Is it clever to create one for the dev environment? Or everyone should create it is own?

File: docker/Dockerfile-api [hunk 2]
```diff
@@ -15,9 +15,11 @@ RUN dnf -y install \
     openssl-devel \
     python3-devel \
     python3-pip \
+    && dnf update -y \
```
Comment (mvalarh) [inline docker/Dockerfile-api:18]: I tried today to install and make it work and i was missing package : `python3-mod_wsgi`. So maybe it is useful to add it

File: docker-compose.yml (hunk 1 above)
Comment (pbortlov) [reply docker-compose.yml]: removed

File: docker/Dockerfile-api (hunk 2 above)
Comment (pbortlov) [reply docker/Dockerfile-api:18]: This is interesting. I don't have a problem with `httpd`. My API is running correctly. Could you share how do you use dev env?

Comment (mvalarh) [reply docker/Dockerfile-api:18]: Strange i cannot reproduce it now. You can ignore it.

File: docker/Dockerfile-workers [hunk 3]
```diff
@@ -58,24 +51,24 @@ RUN dnf -y install \
...
-ADD https://github.com/operator-framework/operator-sdk/releases/download/v1.9.0/operator-sdk_linux_amd64 /usr/bin/operator-sdk
+ADD https://github.com/operator-framework/operator-sdk/releases/download/v1.10.1/operator-sdk_linux_amd64 /usr/bin/operator-sdk
 RUN chmod +x /usr/bin/operator-sdk
 
 # Adjust storage.conf to enable Fuse storage.
 RUN sed -i -e 's|^#mount_program|mount_program|g' /etc/containers/storage.conf
 COPY docker/libpod.conf /usr/share/containers/libpod.conf
 
 COPY . .
+RUN pip3 install --upgrade pip
```
Comment (yashvardhannanavati) [inline docker/Dockerfile-workers:71]: I believe we are already installing pip with dnf. It will install the latest one. This can be removed, no?

Comment (pbortlov) [reply docker/Dockerfile-workers:71]: this is pip upgrade. It should stay

**Review by lipoja — DISMISSED** [review]

File: docker/Dockerfile-workers [hunk 4]
```diff
@@ -23,23 +23,16 @@ RUN dnf -y install \
     libseccomp-devel \
     ostree-devel \
     systemd-devel \
+    && dnf update -y \
     && dnf clean all
 
 ENV OPM_SRC="/src"
-ENV OPM_TAG="v1.17.0"
+ENV OPM_TAG="v1.17.3"
```
Comment (lipoja) [inline docker/Dockerfile-workers:30]: Should we go with v1.18.0?

**Review by midnightercz — APPROVED** [review]

File: docker/Dockerfile-workers (hunk 4 above)
Comment (pbortlov) [reply docker/Dockerfile-workers:30]: I will remove building the OPM and add binary here. Thanks @lipoja

**Review by midnightercz — DISMISSED** [review]

File: docker/Dockerfile-workers (hunk 4 above)
Comment (pbortlov) [reply docker/Dockerfile-workers]: changed

**Review by lipoja — DISMISSED** [review]

File: docker/Dockerfile-api [hunk 5]
```diff
@@ -15,9 +15,12 @@ RUN dnf -y install \
     openssl-devel \
     python3-devel \
     python3-pip \
+    python3-wheel \
+    && dnf update -y \
     && dnf clean all
 COPY . .
 COPY ./docker/iib-httpd.conf /etc/httpd/conf/httpd.conf
+RUN pip3 install --upgrade pip
```
Comment (yashvardhannanavati) [inline docker/Dockerfile-api:23]: Could you please add a comment here explaining the issue

File: docker/Dockerfile-workers (hunk 3 above)
Comment (yashvardhannanavati) [reply docker/Dockerfile-workers:37]: Sounds good. Could you please add a comment here.

Comment (pbortlov) [reply docker/Dockerfile-workers:41]: added the comment

File: docker/Dockerfile-api (hunk 5 above)
Comment (pbortlov) [reply docker/Dockerfile-api:26]: added the comment

**Review by midnightercz — DISMISSED** [review]

**Review by chandwanitulsi — APPROVED** [review]

**Review by lipoja — APPROVED** [review]

**Pull request merged**